crates/plugin_bundle/src/lib.rs

```diff
@@ -48,7 +48,17 @@ impl Plugin for FarmPluginBundle {
 
     resource_pots.sort_by_key(|item| item.id.clone());
 
-    let r = resource_pots.iter().map(|item| &**item).collect::<Vec<_>>();
+    let r = resource_pots
+      .iter()
+      .filter(|item| {
+        if matches!(context.config.output.target_env, TargetEnv::Library) {
+          true
+        } else {
+          matches!(item.resource_pot_type, ResourcePotType::Runtime)
+        }
+      })
```

Comment:
Maybe you can use `matches!(context.config.output.target_env, TargetEnv::Library) || matches!(item.resource_pot_type, ResourcePotType::Runtime)`?

Reply:
@shulandmimi